nipype/interfaces/dcm2nii.py

```diff
@@ -292,7 +292,7 @@ def _parse_stdout(self, stdout):
                         bvals.append(out_file + ".bval")
                         find_b = False
                 # next scan will have bvals/bvecs
-                elif 'DTI gradient directions' in line:
+                elif 'DTI gradients' in line:
```

Comment:
would suggest changing to `elif 'DTI gradient' in line:` to make compatible with former output as well